src/components/RichTextEditor/components/elements/link.tsx

```diff
@@ -0,0 +1,18 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { PlateRenderElementProps } from "@udecode/plate";
+import React from "react";
+
+export const LinkElement = (props: PlateRenderElementProps) => {
```

Comment:
You could destruct and make a direct return 
```suggestion
export const LinkElement: FC<PlateRenderElementProps> = ({ element, attributes, children }) => (
```

Reply:
Refactored